controllers/childrenController.js

```diff
@@ -0,0 +1,40 @@
+const db = require("../models");
```

Comment:
Not suggesting a change... But controller may not be the best word for this file. This is more of an orm type of file. childrenOrm or childrenObjectMapper or ???